WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderPostUiStateBuilder.kt

```diff
@@ -191,8 +191,7 @@ class ReaderPostUiStateBuilder @Inject constructor(
         } else {
             R.string.reader_add_bookmark
         }
-        // TODO malinjir shouldn't the action be disabled just for posts which don't have blog and post id?
-        return if (!post.isDiscoverPost) {
```

Comment:
This should be the only change in the behavior -> I enabled "bookmark" actions for posts in Discover tab if they have valid postid and blogid -> posts without post/blog ids are opened in a browser by default.